modules/aws_account_permissions/outputs.tf

```diff
@@ -0,0 +1,8 @@
+output "role_arn" {
+  value = aws_iam_role.illumio_cloud_integration_role.arn
+}
+
+output "external_id" {
+  value = random_uuid.role_secret.result
+  sensitive = true
+}
```

Comment:
This shouldn't be output. It should be used directly within the module in a `illumio-cloudsecure_aws_account` resource.